pandas/core/frame.py

```diff
@@ -10563,13 +10563,13 @@ def isin(self, values) -> DataFrame:
 
         When ``values`` is a Series or DataFrame the index and column must
         match. Note that 'falcon' does not match based on the number of legs
-        in df2.
+        in other.
 
-        >>> other = pd.DataFrame({'num_legs': [8, 2], 'num_wings': [0, 2]},
+        >>> other = pd.DataFrame({'num_legs': [8, 3], 'num_wings': [0, 2]},
         ...                      index=['spider', 'falcon'])
         >>> df.isin(other)
                 num_legs  num_wings
-        falcon      True       True
+        falcon      False       True
```

Comment:
nitpick can you line these up (as would happen in the terminal)

Reply:
Do you want me to adjust `True` and `False` in line 10572?

Reply:
I think jeff means deleting one whitespace between falcon and false

Reply:
Understood , I am force pushing the suggested changes